### PR TITLE
Backend - Game Routes - Get game by Id

### PR DESCRIPTION
**Pull request by DaraghK93**
Another route here for getting the game by its id

send a GET to http://localhost:3001/api/league/:leagueId

user must be in that league so send correct token

right now, the object being returned is like this
Screenshot 2022-11-29 at 19 00 15

This can change if needs be. @beckskelly, happy to go with whatever you think is best. Currently I'm sending down the valuehistory array as it is, but there is scope there for me to calculate the daily change on the backend to save some processing on the frontend. I can work away at trying to get this out, but think it's important we start linking this with the frontend ASAP.

### CODE REVIEW

**Review by joeycor92 — APPROVED** [review]
Works perfectly for me anyway. I'm happy to approve this. Great work master aggregator.

**Review by WarrenK-design — APPROVED** [review]
Hi Daragh, 

Works for me functionally have tried creating new league and sending request. Joining new league with new user and getting it with this route seems to work too. 

Before you merge though I think wait for @beckskelly to approve and make sure this data ok for leaderboard for the game.

**Review by beckskelly — APPROVED** [review]
Looks good data wise, I will let you know if it needs changing

**Pull request merged**
